catalogs/catalog-hadoop/src/test/java/org/apache/gravitino/catalog/hadoop/integration/test/HadoopOSSCatalogIT.java

```diff
@@ -197,4 +195,11 @@ public void testCreateSchemaAndFilesetWithSpecialLocation() {
     // Delete catalog
     metalake.dropCatalog(localCatalogName, true);
   }
+
+  protected static boolean ossIsPrepare() {
```

Comment:
You'd better using the same method name like "isXXXConfigured".

Reply:
fixed.